sdg/components.py

```diff
@@ -22,20 +22,26 @@ def git_clone_op(
             # Increase logging verbosity
             set -x &&
 
+            # Set Preferred CA Cert
```

Comment:
@gmfrasca , I found a simpler way to set the http.sslCAInfo to be used by git in the git_clone and sdg_tasks.

You can remove all changes made recently in components.py and just set the GIT_SSL_CAINFO env var in pipeline.py. 
```
git_clone_task.set_env_variable("GIT_SSL_CAINFO", "/dsp-custom-certs/dsp-ca.crt")
sdg_task.set_env_variable("GIT_SSL_CAINFO", "/dsp-custom-certs/dsp-ca.crt")
...
I've tested this approach in a disconnected cluster and git was able to clone the repo with SelfSigned certificates. I haven't tested it in connected clusters cloning regular github